build.gradle

```diff
@@ -11,7 +10,6 @@ plugins {
 
 allprojects {
     apply plugin: 'java'
```

Comment:
I guess this plugin should be also removed

Reply:
Right now removal causes some additional problems (spotless is applied to all projects and needs `java` there as well), so we should clean it up after other common scripts are added